Remove the shebang line, so that pkgdepend doesn't get upset and
fail to find this file in the proto area.

Not submitting upstream because this is Solaris packaging-specific.

--- pyScss-1.3.4/scss/tool.py.orig	2018-07-04 18:00:07.836960580 -0700
+++ pyScss-1.3.4/scss/tool.py	2018-07-04 18:00:12.455840261 -0700
@@ -1,6 +1,6 @@
-#!/usr/bin/env python
+#
 from __future__ import absolute_import
 from __future__ import print_function
 
 from collections import deque
 from contextlib import contextmanager
